### PR TITLE
Add Hebrew to language list

### PR DESCRIPTION
**Pull request by McLoo**
Now that Hebrew is 100% translated and pulled from transifex, let's enabled it in the language list.
Shalom.

### CODE REVIEW

File: res/values/arrays.xml [hunk 1]
```diff
@@ -49,6 +50,7 @@
       <item>es</item>
       <item>fr</item>
       <item>el</item>
+      <item>he</item>
```
Comment (moxie0) [inline res/values/arrays.xml]: think it's iw?

Comment (agrajaghh) [inline res/values/arrays.xml]: http://stackoverflow.com/a/8470980/3755928
Android :imp:

**Comment by McLoo** [comment]
@moxie0 that would explain the lang_map [he:iw](https://github.com/WhisperSystems/Signal-Android/blob/master/.tx/config#L3)

According to that stackoverflow discussion, we should go the HE way. If it would be possible to pull hebrew twice, we were on the 100% side if the language preference is set to default. 
(Maybe by defining a second lang_map entry *he:he* alongside the existing *he:iw*)

@amire80 would you mind sharing your expertise on that?

**Comment by amire80** [comment]
Thanks for asking :)

First of all, after the latest upgrade from the Play store, Hebrew strings appear, so whatever you already did does work.

I know a few things about localization expert, but I'm not an Android development expert at all.

I do know that "iw" is an old and deprecated ISO code. "he" is now used almost everywhere, but "iw" somehow made it to the Java libraries that Android uses. Android apps is probably the last area of modern software where it still pops up.

Most of my work is around MediaWiki localization. When we export the translations to the Android app's source repository, it is saved as "iw". These are the files for the Wikipedia Android app: http://git.wikimedia.org/tree/apps%2Fandroid%2Fwikipedia.git/master/app%2Fsrc%2Fmain%2Fres . You can see that it uses values-iw, like Signal does now.

If you can change it to "he" without breaking anything else, it would be great, but don't waste too much time on that. Issue #4714, for example, is much more important for all RTL languages - Hebrew, Arabic, Persian, Urdu, and others.

(And, as I already wrote in #4714, I want to thank you again for making the translation process so smooth.)

**Comment by moxie0** [comment]
@amire80 is there a yiddish translation forthcoming?

**Comment by amire80** [comment]
Not that I know - I don't know Yiddish (unfortunately).

But Arabic, Persian, Urdu and Pashto should be very important (all of them have more speakers than Hebrew).

**Comment by moxie0** [comment]
@amire80 ah, I thought I saw you request yiddish on transifex, got excited

**Comment by McLoo** [comment]
@amire80 thanks for elaborating!

**Comment by McLoo** [comment]
In case no one recognized:
Android (5.1.1) is happy with HE and IW - doubled checked that. 
Odd. Sanely handled though.

**Pull request closed without merging**